app/src/pages/inside/dashboardItemPage/newWidgets/chartConfigs/chartConfigs.js

```diff
@@ -0,0 +1,34 @@
+export const chartConfigs = {
+  colors: {
+    automationBug: '#f7d63e',
+    automation_bug: '#f7d63e',
+    duration: '#507fd5',
+    failed: '#f36c4a',
+    interrupted: '#a94442',
+    invalid: '#ff3222',
+    investigated: '#87b87f',
+    negative: '#e52e2e',
+    noDefect: '#777777',
+    no_defect: '#777777',
```

Comment:
Do we really need to duplicate colors and store both cases?

Reply:
And majority of these colors already defined at `common/constants/` folder, it's will be better to use them.

Reply:
Fixed. Moved colors to common file